data/planning/index.html

```diff
@@ -10,57 +10,356 @@
+      <td>map data</td>
+      <td>AGRC, WFRC, MAG</td>
+      <td>Existing and planned bicycle infrastructure on/near roadways (bike lanes, cycle tracks, bike boulevards etc.), is coded on the road centerline dataset using Bike_L(eft), Bike_R(ight), Bike_Plan_L, Bike_Pln_R attributes. The Bike_RegPr field is used to distinguish regional bike facilities from more locally oriented facilities. This dataset is populated based on local data and local plans compiled by WFRC & MAG, and 2019 aerial photography review by UDOT Traffic and Safety. See Trails and Pathways for non-road bike/ped facilities.</td>
+    </tr>
+    <tr>
+      <td>Active Transportation</td>
+      <td><a href="{% link data/recreation/trails/index.html %}">Trails and Pathways (Bike/Ped/Hike)</a></td>
+      <td>map data</td>
+      <td>AGRC</td>
+      <td>Existing and planned trails and pathways (off-street), statewide. The Status field differentiates planned from existing and Carto_Code and Class differentiate different types of trails and pathways. See On-Street Bike Facilities for bike lanes, cycle tracks, etc.</td>
+    </tr>    
+    <tr>
+      <td>Active Transportation</td>
+      <td><a href="{% link data/transportation/roads-system/index.html %}">Road and Highway System</a></td>
+      <td>map data</td>
+      <td>AGRC, WFRC, MAG</td>
+      <td>Existing and planned bicycle infrastructure on/near roadways is coded on the road centerline dataset using Bike_L(eft), Bike_R(ight), Bike_Plan_L, Bike_Pln_R attributes. The Bike_RegPr field is used to distinguish regional bike facilities from more locally oriented facilities. This dataset is populated based on local data and local plans compiled by WFRC & MAG, and 2019 aerial photography review by UDOT Traffic and Safety.</td>
+    </tr>    
+    <tr>
+      <td>Aerial Photography</td>
+      <td><a href="{% link discover/license/index.md %}">Current High Res Aerial Photography</a></td>
+      <td>map data services</td>
+      <td>AGRC</td>
+      <td>The most current high resolution aerial photography for Utah is made available to GIS users (desktop and webapps) from AGRC's Discover server. This commercially-licensed imagery is available at no charge to state and local government (including tribes, k-12, and colleges), and their contractors and formal partners, for download and as high-performance web services.</td>
+    </tr>
+     <tr>
+      <td>Aerial Photography</td>
+      <td><a href="{% link data/aerial-photography/index.html %}">Archival Aerial Photography</a></td>
+      <td>map data</td>
+      <td>AGRC & UGS</td>
+      <td>This link opens the AGRC aerial photography page with information on accessing older aerial photography including 1990's era DOQ's and older scanned aerial imagery </td>
+    </tr>
+     <tr>
+      <td>Base Maps</td>
+      <td><a href="{% link discover/index.html %}">Base Maps</a></td>
```

Comment:
The anchor can go after the link generation but still inside the href quotes. I'll add it when I get a chance.